db/migrations/20230619170804_rename_collection_to_article_connection.js

```diff
@@ -0,0 +1,14 @@
+const { baseDown } = require('../utils')
+
+const newName = 'article_connection'
```

Comment:
do we have any services rely on `collection`? e.g. analytics

Reply:
might be; but no big deal